packages/ui/src/views/canvas/CredentialInputHandler.jsx

```diff
@@ -100,6 +104,7 @@ const CredentialInputHandler = ({ inputParam, data, onSelect, disabled = false }
                         <div key={reloadTimestamp} style={{ display: 'flex', flexDirection: 'row' }}>
                             <AsyncDropdown
                                 disabled={disabled}
+                                key={`${data.id}-${inputParam.name}-${data.inputs[credentialId]}`}
```

Comment:
why do we need this?

I got error when trying to create a custom assistant:
![image](https://github.com/user-attachments/assets/393e7a85-a5a6-4d91-99c3-c00a9d4bf784)